src/google/protobuf/compiler/java/map_field.cc

```diff
@@ -792,7 +792,7 @@ void ImmutableMapFieldGenerator::GenerateKotlinDslMembers(
       variables_,
       "@kotlin.jvm.JvmSynthetic\n"
       "@JvmName(\"clear$kt_capitalized_name$\")\n"
-      "fun com.google.protobuf.kotlin.DslMap"
+      "public fun com.google.protobuf.kotlin.DslMap"
```

Comment:
@deannagarcia FYI, this is a change which was skipped for some reason in #11068 when cherry-picking.